src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildAdapter.kt

```diff
@@ -52,6 +54,15 @@ class CargoBuildAdapter(
             .withExecutionFilters(*createFilters(context.cargoProject).toTypedArray())
             .withRestartAction(createRerunAction(processHandler, context.environment))
             .withRestartAction(createStopAction(processHandler))
+            .withContentDescriptorSupplier {
+                val activateToolWindow = context.environment.isActivateToolWindowBeforeRun
+                BuildContentDescriptor(null, null, object : JComponent() {}, "Build")
+                    .apply {
+                        isActivateToolWindowWhenAdded = activateToolWindow
+                        isActivateToolWindowWhenFailed = activateToolWindow
+                        isAutoFocusContent = activateToolWindow
```

Comment:
I don't think it's a good idea to steal focus on running cargo command, it may be rather annoying especially when you launch build/run configuration via shortcut